src/test/lib/concurrency/stress_test.cpp

```diff
@@ -54,6 +54,8 @@ TEST_F(StressTest, TestTransactionConflicts) {
   };
 
   // Create the async objects and spawn them asynchronously (i.e., as their own threads)
+  // Note that async has a bunch of issues (cf. Mastering the C++17 STL, pages 205f).
```

Comment:
Hm, I don't really know what to make of that and if the reader of this comment will order the book to understand the comment. Is there an online version of it?

Reply:
> Note that async has a bunch of issues:
> - https://stackoverflow.com/questions/12508653/what-is-the-issue-with-stdasync
> - Mastering the C++17 STL, pages 205f

better?